### ISSUE TITLE
No way to add multiple labels without duplicating test lists?

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/wpt-metadata#2308]
I'd like to have both the `interop-2022-rfc` and `interop-2022-forms` labels in a bunch of places, like here:

https://github.com/web-platform-tests/wpt-metadata/blob/233aab4846cedb5d1308ff6db2bce137085bd2f6/html/semantics/forms/constraints/META.yml#L19-L41

It looks like the only way to do this is to duplicate the whole block. I would like to be able to do this:
```yaml
    - label: ['interop-2022-rfc', 'interop-2022-forms']
      results:
        - test: ...
```

This would really only work when editing the YAML by hand however, or the code for updating YAML would need to check if an existing label labels exactly the same tests and then merge the blocks.

@KyleJu WDYT?

**Comment by KyleJu** [comment]
I can see this being useful if each test tends to share 2+ labels. There would be some tradeoffs between not duplicating tests and slightly more complicated merge logic.  Also when we triage a new test, they are more likely to be on a separate `link` entry than an existing one since existing ones have multiple labels applied to them.

**Comment by foolip** [comment]
I'm not totally sure we'll need this any longer, we might just go with one label per focus area. But https://github.com/web-platform-tests/wpt-metadata/pull/2320 just ran into a similar problem when trying to label tests in the same directory. Since the URL is considered to be "" for both, it failed the tests.

**Comment by KyleJu** [comment]
> I'm not totally sure we'll need this any longer, we might just go with one label per focus area. But #2320 just ran into a similar problem when trying to label tests in the same directory. Since the URL is considered to be "" for both, it failed the tests.

This issue should be fixed by #2321
